lib/iex/lib/iex/pry.ex

```diff
@@ -482,7 +482,7 @@ defmodule IEx.Pry do
 
     # Have an extra binding per argument for case matching.
     case_vars =
-      for id <- tl(Enum.to_list(0..length(args))) do
+      for id <- tl(Enum.to_list(1..(length(args) + 1))) do
```

Comment:
This change is incorrect. We were skipping the zero already due to the `tl` call. With this change, it will start counting from 2.

Reply:
Yes, there was no test. I was waiting for your input. I will revert it.